### ISSUE TITLE
JinaD Flow with `replicas>1` & `parallels>1` not supported

### ISSUE THREAD

**Issue by deepankarm** [jina-ai/jina#3317]
**Steps to reproduce**
A Flow in JinaD with a config like below will have all requests blocked. Note that parallels: 1 will work in this case. More about this issue.

```
jtype: Flow
version: '1'
pods:
  - uses: docker://query-executor
    replicas: 2
    parallel: 2 # anything more than 1
```

**Comment by JoanFM** [comment]
Hey @deepankarm ,

Is this issue solved by #3469 ?

**Comment by JoanFM** [comment]
Closing this issue as it should be handled by https://github.com/jina-ai/internal-tasks/issues/271
